Source/Kesmai.Server/Game/Items/Equipment/Weapons/Weapon.cs

```diff
@@ -53,32 +77,47 @@ public override int CalculateBlockingBonus(ItemEntity item)
 			if (item is IWeapon weapon)
 				flags = weapon.Flags;
 
-			var blockingBonus = 0;
+			var bonusBlock = 0;
 
 			if ((flags & WeaponFlags.Projectile) != 0)
 			{
 				return ProjectileProtection;
 			}
 			else
 			{
-				var itemProtections = new List<int>();
+				bonusBlock += GetMeleeBonus(item, flags)
+			}
 
-				if ((flags & WeaponFlags.Piercing) != 0)
-					itemProtections.Add(PiercingProtection);
+			return bonusBlock + BaseArmorBonus;
+		}
+		
+		/// <inheritdoc/>
+		/// <remarks>
+		/// Weapons only provide a blocking bonus against other weapons. Two-handed weapons
+		/// should only provide a benefit if equipped in the right-hand.
+		/// Used for mitigation calculation. Weapon BaseArmorBonus does not help mitigate.
+		/// </remarks>
+		public int GetWeaponBonus(ItemEntity item)
+		{
+			var flags = WeaponFlags.Bashing;
 
-				if ((flags & WeaponFlags.Slashing) != 0)
-					itemProtections.Add(SlashingProtection);
+			if (item is IWeapon weapon)
+				flags = weapon.Flags;
 
-				if ((flags & WeaponFlags.Bashing) != 0)
-					itemProtections.Add(BashingProtection);
+			var weaponBonus = 0;
 
-				if (itemProtections.Any())
-					blockingBonus += itemProtections.Min();
+			if ((flags & WeaponFlags.Projectile) != 0)
+			{
+				weaponBonus += ProjectileProtection;
+			}
+			else
+			{
+				weaponBonus += GetMeleeBonus(item, flags)
 			}
 
-			return blockingBonus + BaseArmorBonus;
+			return weaponBonus;
```

Comment:
this seems kind of inconsistent to me, If we are going to add the mitigation in there might as well be consistent,  Adding it in though might require some balance changes overall though (and adding in gauntlets).

Reply:
That there was because I didn't think something like the BBS should provide 3 damage mitigation. Also why I broke up the DR and Mitigation portions...to make things more consistent